frontend/app/routes/$lang/_protected/alerts/index.tsx

```diff
@@ -12,22 +14,24 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   featureEnabled('email-alerts');
 
   const raoidcService = await getRaoidcService();
-
   await raoidcService.handleSessionValidation(request, session);
-
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
 
+  const userService = getUserService();
+  const subscriptionService = getSubscriptionService();
   invariant(userInfoToken.sin, 'Expected userInfoToken.sin to be defined');
 
-  return redirect(getPathById('$lang/_protected/alerts/subscribe/index', params));
-
-  //TODO fix the redirection once we can retrieve the status of the subscription
+  const userInfo = userService.getUser(userInfoToken.sub);
+  const subscriptions = await subscriptionService.getSubscription(userInfoToken.sub);
+  const emailVerified = (await userInfo).emailVerified;
 
-  /*if (!alertSubscription || alertSubscription.registered === false) {
-    return redirect(getPathById('$lang/_protected/alerts/subscribe/index', params));
+  if (emailVerified === false && !subscriptions) {
+    return redirect(getPathById('$lang/_protected/alerts+/subscribe+/index', params));
   }
-  if (alertSubscription.subscribed === true) {
-    return redirect(getPathById('$lang/_protected/alerts/manage/index', params));
+
+  if (emailVerified) {
+    return redirect(getPathById('$lang/_protected/alerts+/manage+/index', params));
   }
-  return redirect(getPathById('$lang/_protected/alerts/subscribe/confirm', params));*/
+
+  return redirect(getPathById('$lang/_protected/alerts+/subscribe+/confirm', params));
```

Comment:
The `+` are no longer needed in the paths as a result of https://github.com/DTS-STN/canadian-dental-care-plan/pull/1526

```suggestion
  return redirect(getPathById('$lang/_protected/alerts/subscribe/confirm', params));
```